AvansDevOps/BacklogItemState/IBacklogItemState.cs

```diff
@@ -10,6 +10,8 @@ public interface IBacklogItemState
     {
         public void StartTask();
         public void FinishTask();
+        public void TestTask();
```

Comment:
Shouldnt the testTask have a success value to know wether the test has succeeded or failed?

Reply:
That is possible, but currently you always go to the TestedState, from there you call the invalidate test/ retest / close test functions to decide if the test failed of not

Reply:
But the ticket should be tested when it is ready for testing right? And if it is successful it should go to tested, if it is not successful it should go back to todo. The invalidate is called when the lead dev disapproves the change and if it is approved it should close. Those two should be on the tested state right? Because that is the step after it. At least that is what is in the flow right now.